sdk/examples/client/02_address_balance.rs

```diff
@@ -43,12 +43,13 @@ async fn main() -> Result<()> {
 
     // Get output ids of outputs that can be controlled by this address without further unlock constraints
     let output_ids_response = client
-        .basic_output_ids([
-            BasicOutputsQueryParameters::Address(first_address.clone()),
-            BasicOutputsQueryParameters::HasExpiration(false),
-            BasicOutputsQueryParameters::HasTimelock(false),
-            BasicOutputsQueryParameters::HasStorageDepositReturn(false),
-        ])
+        .basic_output_ids(BasicOutputsQueryParameters {
```

Comment:
builder pattern?

Reply:
Like this?
```
BasicOutputsQueryParametersBuilder::default()
                .with_address(first_address.clone())
                .with_has_expiration(false)
                .with_has_timelock(false)
                .with_has_storage_deposit_return(false)
                .build()?
```
Can do it with derive_builder, but don't really see the benefit

Reply:
Yeah that's what I was thinking. The benefit isn't huge, true. I don't mind really.

Reply:
Although if this is user-facing it may be worth it just for convenience.